CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## unrelease
+
+### Added
+- You can now use regexps for field names:
+  ```
+  var $X = {"=~/[lL]ocation/": $Y};
```

Comment:
Is there a reason we went with `=~` as syntax to use regex?

Reply:
I'm guessing an very old design choice. All other regexes work like this https://docs.semgrep.dev/config/simple#string-matching so this is more of a bugfix PR to allow it on keys

Reply:
it comes from Perl, which uses this =~ operator. Perl is the regexp language :)

Reply:
what do you propose?

Reply:
https://xkcd.com/208/

Reply:
Just wondering. It's not the best from the standpoint of a new user coming in but if it exists elsewhere then it's easier to justify